cluster/gce/gci/configure-helper.sh

```diff
@@ -1288,7 +1288,7 @@ function start-kube-apiserver {
   fi
 
 
-  local authorization_mode="RBAC"
+  local authorization_mode="Node,RBAC"
```

Comment:
According to the [OWNERS file](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/container-linux/OWNERS), it's @euank, @yifan-gu, and @ethernetdan.